docs/README.md

```diff
@@ -5,6 +5,6 @@ We currently have the following components available for Angular applications:
 - [Empty State](https://github.com/pxblue/angular-component-library/blob/dev/docs/EmptyState.md)
 - [Hero & Hero Banner](https://github.com/pxblue/angular-component-library/blob/dev/docs/Hero.md)
 - [Info List Item](https://github.com/pxblue/angular-component-library/blob/dev/docs/InfoListItem.md)
+- [ListItemTag](https://github.com/pxblue/angular-component-library/blob/dev/docs/ListItemTag.md)
```

Comment:
Please also update the changelog to include the addition of the listitemtag component

Reply:
That should be in there.  I had included it in between InfoListItem and Scorecard

Reply:
Need space between the words, `List Item Tag`